docs/http/http-metrics.md

```diff
@@ -74,7 +70,7 @@ of `[ 0, 0.005, 0.01, 0.025, 0.05, 0.075, 0.1, 0.25, 0.5, 0.75, 1, 2.5, 5, 7.5,
 <!-- semconv metric.http.server.duration(full) -->
 | Attribute  | Type | Description  | Examples  | Requirement Level |
 |---|---|---|---|---|
-| `http.route` | string | The matched route (path template in the format used by the respective server framework). See note below [1] | `/users/:userID?`; `{controller}/{action}/{id?}` | Conditionally Required: If and only if it's available |
+| `http.route` | string | The matched route. For example, the path template in the format used by the respective server framework. See note below [1] | `/users/:userID?`; `{controller}/{action}/{id?}` | Conditionally Required: If and only if it's available |
```

Comment:
Suggest a better change and I'll apply it in the other two places.

Reply:
These are generated from YAML so we need to update the source https://github.com/open-telemetry/semantic-conventions/blob/c6efa4b2b44f9950b332a46311834e23b85bf979/model/http-common.yaml#L109 and regenerate the Markdown files (refer to https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#yaml-to-markdown).

Reply:
> These are generated from YAML so we need to update the source
> 
> https://github.com/open-telemetry/semantic-conventions/blob/c6efa4b2b44f9950b332a46311834e23b85bf979/model/http-common.yaml#L109
> 
> and regenerate the Markdown files (refer to https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#yaml-to-markdown).

I'm pegged on my real job. If you can point me to the markdown files and help me get started I can migrate my changes.